dashboard/src/app/core/entities/territory/territory.ts

```diff
@@ -161,27 +142,12 @@ export class Territory
     if (formValues.company_id) this.company_id = formValues.company_id;
     else delete this.company_id;
 
-    if (formValues.children && (formValues.format === 'parent' || formValues.format === 'insee'))
-      this.children = formValues.children;
-    else delete this.children;
+    this.children = formValues.children;
 
     this.ui_status = {};
```

Comment:
Try to remove this completly

Reply:
On fait ça dans un prochain merge ?

Reply:
Yes j'ai commencé ici => https://github.com/betagouv/preuve-covoiturage/tree/refactor/cleanup_territory_models